data_steward/analytics/cdr_ops/ac120_cpt_modifiers.py

```diff
@@ -71,7 +71,7 @@ def cstr(s, color='black'):
 
 
-'''.format(DATASET_ID=parameters.DEID_DATASET_ID)
+'''.format(DATASET_ID=parameters.UNION_DATASET_ID)
```

Comment:
You and I found it helpful to swap the dataset to run queries on. Rather than specifying a particular dataset in each query, it might make it easier to use a variable which is declared in the beginning of the notebook (e.g. `dataset_id = parameters.DEID_DATASET_ID`).

Reply:
yes definitely, will make that adjustment.